internal/command/command_plugin.go

```diff
@@ -108,6 +109,7 @@ func (cp *CommandPlugin) createConnection(ctx context.Context, resource *mpi.Res
 		slog.ErrorContext(ctx, "Unable to create connection", "error", err)
 	}
 	if createConnectionResponse != nil {
+		cp.commandService.StartSubscription()
```

Comment:
Thinks its better to create the context and pass it to the function here.
Also you could call `go cp.commandService.subscribe(subscribeContext)` instead of having the StartSubscription function just wrapping the subscribe function.